src/test/java/net/minecraftforge/debug/entity/player/PlayerSpawnPhantomsEventTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.entity.player;
+
+import net.minecraftforge.event.entity.player.PlayerSpawnPhantomsEvent;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("player_spawn_phantoms_event_test")
+@Mod.EventBusSubscriber()
+public class PlayerSpawnPhantomsEventTest
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test.

Also, we should probably make the test mod result more obvious, like setting the phantoms spawned to a unnaturally large amount, instead of preventing spawns (which may be attributed to luck).